chemprop/cli/hpopt.py

```diff
@@ -46,15 +46,15 @@
         "activation": tune.choice(categories=list(Activation.keys())),
         "aggregation": tune.choice(categories=list(AggregationRegistry.keys())),
         "aggregation_norm": tune.quniform(lower=1, upper=200, q=1),
-        "batch_size": tune.loguniform(lower=16, upper=256, base=2),
-        "depth": tune.quniform(lower=2, upper=6, q=1),
+        "batch_size": tune.choice(np.logspace(start=4, stop=8, num=5, base=2, dtype=int)),
```

Comment:
I am thinking about just using `tune.choice([16, 32, 64, 128, 256])`

Reply:
Yes, the current line is equivalent. logspace() outputs the [16, 32, 64, 128, 256] vector.
Let me know if you still think its not equivalent.

Reply:
I meant replace `np.logspace(start=4, stop=8, num=5, base=2, dtype=int)` with `[16, 32, 64, 128, 256]`. I think it is clearer and there is no downside to using this.

Reply:
I think the current implementation is better

Reply:
I agree with Shih-Cheng that `[16, 32, 64, 128, 256]` is better than using `np.logspace` because it is immediately obvious what the choices are. With `np.logspace` a code reader has to either run the line of code (as I did when reviewing this before) or know what `np.logspace` does and their powers of two. 

I could see the current implementation being better if we expect to change the choices significantly in the future as it is more flexible, but that flexibility probably isn't need.